package com.facebook.ads.redexgen.X;

import android.graphics.drawable.GradientDrawable;
import android.text.TextUtils;
import android.view.View;
import android.view.animation.DecelerateInterpolator;
import android.widget.ImageView;
import android.widget.LinearLayout;
import android.widget.TextView;
import com.facebook.internal.C2372q;
import o.C7193j50;

/* renamed from: com.facebook.ads.redexgen.X.dq  reason: case insensitive filesystem */
/* loaded from: assets/audience_network/classes2.dex */
public final class C1496dq extends LinearLayout {
    public LinearLayout A00;
    public String A01;
    public boolean A02;
    public final int A03;
    public final C1673gi A04;
    public final C1367bk A05;
    public final C1373bq A06;
    public static String[] A07 = {"mO6OCzz183u1Bigulq5KeRnnu3UhUErY", "M8wuVwCqg47swjluxOTBwYkXCALJOS1f", "isO3a6yeUrEqJHaKmCFaV", "J8y0fQZRtVpgz1Itec1cmA3ue79eN13N", "z6IigVxY2j2waSprQRCf4749VcxJv5uz", "GG3OTiE6t7MwpY", "UBA0QA1YjTXejDlGkh9vREu7YbW4o1hK", "iNB15eFG13fee2"};
    public static final int A0A = (int) (XX.A02 * 16.0f);
    public static final int A0B = (int) (XX.A02 * 32.0f);
    public static final int A0C = (int) (XX.A02 * 16.0f);
    public static final int A09 = (int) (XX.A02 * 4.0f);
    public static final int A08 = (int) (XX.A02 * 72.0f);

    public C1496dq(Cdo cdo) {
        super(Cdo.A04(cdo));
        this.A02 = false;
        this.A01 = "";
        this.A04 = Cdo.A04(cdo);
        this.A05 = new C1367bk(this.A04);
        this.A06 = new C1373bq(this.A04, Cdo.A02(cdo), true, false, true);
        this.A03 = Cdo.A00(cdo);
        this.A02 = Cdo.A08(cdo);
        this.A01 = Cdo.A05(cdo);
        A03(cdo);
    }

    public /* synthetic */ C1496dq(Cdo cdo, GX gx) {
        this(cdo);
    }

    private void A00() {
        A01(this.A05, 150);
        A01(this.A06, C7193j50.f);
        if (this.A00 != null) {
            A01(this.A00, C2372q.m);
        }
    }

    private void A01(View view, int i) {
        view.setTranslationY(i);
        view.setScaleY(0.75f);
        view.setScaleX(0.75f);
        view.animate().translationYBy(-i).scaleX(1.0f).scaleY(1.0f).setDuration(300L).setInterpolator(new DecelerateInterpolator(2.0f));
    }

    private void A02(Cdo cdo) {
        if (!TextUtils.isEmpty(Cdo.A06(cdo))) {
            this.A00 = new LinearLayout(getContext());
            this.A00.setGravity(17);
            int i = A0C / 2;
            if (this.A02) {
                i = 0;
            }
            LinearLayout linearLayout = this.A00;
            int i2 = A0C;
            int i3 = A0C;
            int marginTop = A0C;
            linearLayout.setPadding(i2, i, i3, marginTop / 2);
            LinearLayout.LayoutParams layoutParams = new LinearLayout.LayoutParams(-2, -2);
            layoutParams.setMargins(0, i, 0, 0);
            TextView textView = new TextView(getContext());
            textView.setTextColor(-1);
            YB.A0a(textView, false, 16);
            textView.setText(Cdo.A06(cdo));
            LinearLayout.LayoutParams layoutParams2 = new LinearLayout.LayoutParams(-2, -2);
            ImageView imageView = new ImageView(getContext());
            new LM(imageView, this.A04).A04().A07(Cdo.A07(cdo));
            int i4 = A0A;
            int marginTop2 = A0A;
            LinearLayout.LayoutParams informativeContainerParams = new LinearLayout.LayoutParams(i4, marginTop2);
            int marginTop3 = A0C;
            informativeContainerParams.setMargins(0, 0, marginTop3 / 2, 0);
            if (this.A02) {
                C1361be c1361be = new C1361be(getContext(), 0, -1, YM.PLAYABLE_ICON);
                c1361be.setLayoutParams(new LinearLayout.LayoutParams(A0B, A0B));
                this.A00.addView(c1361be);
            } else {
                this.A00.addView(imageView, informativeContainerParams);
                this.A00.addView(textView, layoutParams2);
                GradientDrawable gradientDrawable = new GradientDrawable();
                gradientDrawable.setCornerRadius(100.0f);
                gradientDrawable.setColor(469762047);
                YB.A0V(this.A00, gradientDrawable);
            }
            View view = this.A00;
            String[] strArr = A07;
            String str = strArr[5];
            String str2 = strArr[7];
            int length = str.length();
            int marginTop4 = str2.length();
            if (length != marginTop4) {
                throw new RuntimeException();
            }
            String[] strArr2 = A07;
            strArr2[0] = "6JdK2pyeEPUWkfGuYxPy07w26kgdYt9J";
            strArr2[1] = "5VEwQONd4c8VI19uC0Ib5tVL9YvIxUH6";
            addView(view, layoutParams);
        }
    }

    private void A03(Cdo cdo) {
        YB.A0N(this.A05, 0);
        this.A05.setRadius(50);
        if (this.A02) {
            this.A05.setRadius(A09);
        } else if (Cdo.A01(cdo).A00() == NW.A05) {
            this.A05.setFullCircleCorners(true);
        } else {
            C1367bk c1367bk = this.A05;
            int i = A09;
            String[] strArr = A07;
            if (strArr[5].length() != strArr[7].length()) {
                throw new RuntimeException();
            }
            A07[2] = "avcxrBXnTsgV2I5Ie1k2l";
            c1367bk.setRadius(i);
        }
        new LM(this.A05, this.A04).A04().A07(Cdo.A03(cdo).A01());
        this.A06.A04(Cdo.A01(cdo).A0G(), Cdo.A03(cdo).A03(), null, false, true);
        if (!this.A02) {
            this.A06.getDescriptionTextView().setAlpha(0.8f);
        } else {
            this.A06.getDescriptionTextView().setText(this.A01);
        }
        this.A06.setAlignment(17);
        LinearLayout.LayoutParams layoutParams = new LinearLayout.LayoutParams(-2, -2);
        layoutParams.setMargins(0, A0C, 0, A0C / 2);
        View view = this.A05;
        LinearLayout.LayoutParams titleAndDescriptionParams = new LinearLayout.LayoutParams(A08, A08);
        addView(view, titleAndDescriptionParams);
        addView(this.A06, layoutParams);
        A02(cdo);
        YB.A0N(this, -14473425);
        setGravity(17);
        setOrientation(1);
    }

    public final void A04(InterfaceC1495dp interfaceC1495dp) {
        A00();
        postDelayed(new GX(this, interfaceC1495dp), this.A03);
    }
}
